docs/blog/newsletter-march-2020.rst

```diff
@@ -0,0 +1,119 @@
+.. post:: March 05, 2020
+   :tags: newsletter
+
+######################################
+Write the Docs Newsletter – March 2020
+######################################
+
+Hello everyone! It's your friendly neighbourhood newsletter team, here with the March edition.
+
+Our first bit of news: we're happy to announce the `results of last year's salary survey </surveys/salary-survey/2019/>`__! Openness is a core value of this community, so it's awesome to be able to present the anonymized data for your purusing.
+
+On the conference side, it's super exciting to see the `roster of speakers for Portland </conf/portland/2020/news/announcing-speakers/#full-speaker-line-up>`__ announced a few weeks ago. Some seriously great talks to look forward to.
+
+We do know you'll have questions about whether the conference is going ahead. Currently, *we have no plans to cancel*. We will be watching the COVID-19 situation closely over the next few weeks, and taking advice from the Oregon health authorities and CDC as it comes, but currently everything is planned to go ahead as normal.
```

Comment:
I'd like to get a short post up on the NA conf site as well about this and link to it this week. How about this verbiage? 

```suggestion
We wanted to also address the COVID-19 situation. Currently, *we have no plans to cancel* the Portland conference, but are planning to take additional precautions during the on-site events. We will be watching the situation closely over the next few weeks, and taking advice from the Oregon health authorities and United States CDC as it comes. We will assess the situation again next week and provide updates as the situation changes. 
```

Reply:
:+1: to this. Maybe with a link to the blog tag for visibility of updates?